sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/parser/AstBuilder.scala

```diff
@@ -4408,16 +4408,6 @@ class AstBuilder extends DataTypeAstBuilder
     }
   }
 
-  /**
-   * Create a [[ShowNamespaces]] command.
-   */
-  override def visitShowNamespaces(ctx: ShowNamespacesContext): LogicalPlan = withOrigin(ctx) {
-    val multiPart = Option(ctx.multipartIdentifier).map(visitMultipartIdentifier)
-    ShowNamespaces(
```

Comment:
I guess users application might depend on ShowNamespaces. Do we need the migration guide if we remove it ? @cloud-fan

Reply:
i can put it back as deprecated, whats the recommendation @cloud-fan ?

Reply:
catalyst plans are not public APIs.

Reply:
OK.